### DISCUSSION TITLE
Splitting data between steady and outlier

### DISCUSSION THREAD

**Discussion by pheer** [TDAmeritrade/stumpy#647] [Help: Coding & Implementations]
Hi I'm new to stumpy - thanks for sharing with community!

Currently analyzing server data. the dataset reports "hours running" each hour. e.g.Server typeA had 10 running hours at hour0 (midnight) in Jan 1,2022 and 12 running hours at hour1 in Jan 1,2022.  If serverA has 10 running hours for all 24 hours in a day..that means 10 serverAs were on for the entire day. One pattern is to see 100 servers run for 4 hours in a day and then return to the baseline of 10.  Goal is to pull out what is steady and what is non-steady server use.  Need way to detect both patterns. ServerA is steady usage at 10 servers per day for entire month or perhaps ServerB has 100 servers run only on Saturdays, etc. 

Any points on getting started with this? Is stumpy good for this type of pattern detection? 

Thanks in advance.

**Comment by JaKasb** [comment]
Your text is hard to understand.
Can you provide visualization/plots of the time series and patterns ?

If yout metric is a cumulative quantity (hours ran since midnight), then it may be a good idea to normalize the metric into a non-cumulative form. eg by taking the first order difference / gradient by the time axis.
